content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -314,18 +315,17 @@ Server:  Message Received
 ```
 4.The server receives and simultaneously shows;
```

Comment:
Change:

> 4.The server receives and simultaneously shows;

To:

> 4. The server receives and simultaneously shows:

Added a missing space. Also a colon (`:`) is more appropriate here than a semi-colon (`;`)